src/main/java/core/basesyntax/service/operations/ReturnOperation.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.service.operations;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+
+public class ReturnOperation implements OperationHandler {
+    @Override
+    public void handle(FruitTransaction transaction) {
+        Storage.getAllFruits().put(transaction.getFruit(), Storage.getAllFruits()
+                .getOrDefault(transaction.getFruit(), 0) + transaction.getQuantity());
```

Comment:
```suggestion
        Storage.getAllFruits().merge(transaction.getFruit(), transaction.getQuantity(), Integer::sum);
```
try smth like this in your handlers